packages/image_picker/image_picker_ios/ios/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array/>
+	<key>NSPrivacyCollectedDataTypes</key>
```

Comment:
> Only shared_preferences has a non-empty manifest, as it is our only plugin that uses a required reason API, and none of our plugins themselves collect private data. 

Wouldn't plugin like image_picker_ios be using `NSPrivacyCollectedDataTypePhotosorVideos` for collecting "The user's photos or videos"?

Reply:
The "Describing data use in privacy manifests" page never defines the term "collect", so currently we're assuming it's the same as the definition Apple gives [for existing privacy declarations](https://developer.apple.com/app-store/app-privacy-details/#data-collection), which is:

> “Collect” refers to transmitting data off the device in a way that allows you and/or your third-party partners to access it for a period longer than what is necessary to service the transmitted request in real time.

Our plugin does not collect any data under this definition. Plugin *clients* may or may not (under this definition) collect data using the plugin, but if they are doing such collection they would need to declare that at the app level since that's where the collection would be.

Reply:
I see, okay makes sense. Thanks for clarifying.